### PR TITLE
Fixed README for .NET 9

### PR DESCRIPTION
**Pull request by raselldev**
### Issues Fixed
README.md

Fixes #
Adding news section of .NET 9

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,6 +20,7 @@
 
 ## Current News
 * November 14, 2023 - [Announcing .NET MAUI in .NET 8](https://devblogs.microsoft.com/dotnet/announcing-dotnet-maui-in-dotnet-8/)
+* November 8, 2024 - [Announcing .NET MAUI in .NET 9](https://learn.microsoft.com/en-us/dotnet/maui/whats-new/dotnet-9?view=net-maui-9.0)
```
Comment (jsuarezruiz) [inline README.md:23]: @davidortinau Do we want to just leave the latest big release blog post (.NET 9 now)?

Comment (jfversluis) [inline README.md:23]:
```suggestion
* November 13, 2024 - [What's new in .NET MAUI for .NET 9](https://learn.microsoft.com/dotnet/maui/whats-new/dotnet-9)
* November 12, 2024 - [Announcing .NET 9](https://devblogs.microsoft.com/dotnet/announcing-dotnet-9/#.net-maui-–-enhancing-multi-platform-app-development)
```

Let's just remove the .NET 8 link and besides the "whats changed" link also add the announcing .NET 9 blog post

File: README.md [hunk 2]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (jfversluis) [inline README.md:1]:
```suggestion
```

**Review by jfversluis — APPROVED** [review]

**Comment by jfversluis** [comment]
Thanks @raselldev ! Have fun with .NET MAUI on .NET 9!

**Pull request merged**